### ISSUE TITLE
email-editor doesn't work

### ISSUE THREAD

**Issue by frontendjorge** [unlayer/angular-email-editor#1]
ERROR in src/app/app.module.ts(5,35): error TS2307: Cannot find module 'email-editor'.

**Comment by frontendjorge** [comment]
solved.... ng build firts

**Comment by frontendjorge** [comment]
No...I'm a lier.... still  TS2307: Cannot find module 'email-editor'.

---------------------------------------------
PACKAGE.JSON
-------------------------------------------------
APP.MODULE.TS
//import email-editor for newsletter
import { EmailEditorModule } from '**email-editor**';
----------------------------------------------------
{
  "name": "sdfdfdff_myproject",
  "version": "0.0.0",
  "license": "MIT",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build --prod",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "^5.2.0",
    "@angular/common": "^5.2.0",
    "@angular/compiler": "^5.2.0",
    "@angular/core": "^5.2.0",
    "@angular/forms": "^5.2.0",
    "@angular/http": "^5.2.0",
    "@angular/platform-browser": "^5.2.0",
    "@angular/platform-browser-dynamic": "^5.2.0",
    "@angular/router": "^5.2.0",
    "@types/moment": "^2.13.0",
    "angular2-permission": "^0.1.3",
    "bootstrap-material-datetimepicker": "^2.7.3",
    "classlist.js": "^1.1.20150312",
    "core-js": "^2.4.1",
    "datepicker": "0.0.0",
    "email-editor": "^1.0.0",
    "gulp-install": "^1.1.0",
    "highlight.js": "^9.13.1",
    "lodash": "^4.17.11",
    "minimatch": "^3.0.4",
    "moment": "^2.20.1",
    "mydatepicker": "^2.6.3",
    "ng2-datepicker": "^2.2.5",
    "rxjs": "^5.5.6",
    "zone.js": "^0.8.19"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.7.0",
    "@angular/cli": "^6.1.3",
    "@angular/compiler-cli": "^5.2.0",
    "@angular/language-service": "^5.2.0",
    "@types/jasmine": "~2.8.3",
    "@types/jasminewd2": "~2.0.2",
    "@types/node": "~6.0.60",
    "codelyzer": "^4.0.1",
    "jasmine-core": "~2.8.0",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "^3.0.0",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "^1.2.1",
    "karma-jasmine": "~1.1.0",
    "karma-jasmine-html-reporter": "^0.2.2",
    "protractor": "^5.4.0",
    "ts-node": "~4.1.0",
    "tslint": "~5.9.1",
    "typescript": "~2.5.3"
  }
}

**Comment by mbagliojr** [comment]
ng build isn't fixing it for me

**Comment by abedev31** [comment]
HI,
It's seems that a wrong import definition into src/app/app.module.ts file.

Replace import component by module, following this:

import { EmailEditorComponent } from '../../projects/email-editor/src/lib/email-editor.component';
must be: 
import { **EmailEditorModule** } from '../../projects/email-editor/src/lib/email-editor.**module**';

and 

....
  imports: [
    BrowserModule,
    **EmailEditorModule**
  ],

**Comment by omprakash-sit** [comment]
I having similar issue "HtmlEditorModule" not found, after ng build module import issue resolved but getting error 
    "export 'defineInjectable' was not found in '@angular/core'
my package.json file is
    "dependencies": {
    "@angular/animations": "5.2.0",
    "@angular/cdk": "5.2.5",
    "@angular/common": "5.2.0",
    "@angular/compiler": "5.2.0",
    "@angular/core": "5.2.0",
    "@angular/forms": "5.2.0",
    "@angular/material": "5.2.5",
    "@angular/platform-browser": "5.2.0",
    "@angular/platform-browser-dynamic": "5.2.0",
    "@angular/router": "5.2.0",
    "@ng-bootstrap/ng-bootstrap": "1.0.0",
    "angular-email-editor": "^0.7.1",
    }

**Comment by kaustubhkhare19** [comment]
> I having similar issue "HtmlEditorModule" not found, after ng build module import issue resolved but getting error
> "export 'defineInjectable' was not found in '@angular/core'
> my package.json file is
> "dependencies": {
> "@angular/animations": "5.2.0",
> "@angular/cdk": "5.2.5",
> "@angular/common": "5.2.0",
> "@angular/compiler": "5.2.0",
> "@angular/core": "5.2.0",
> "@angular/forms": "5.2.0",
> "@angular/material": "5.2.5",
> "@angular/platform-browser": "5.2.0",
> "@angular/platform-browser-dynamic": "5.2.0",
> "@angular/router": "5.2.0",
> "@ng-bootstrap/ng-bootstrap": "1.0.0",
> "angular-email-editor": "^0.7.1",
> }

@omprakash-sit I had the same issue. angular-email-editor works with angular 7+. I had to upgrade my angular version from 5 to 7.

**Comment by kishorkharade24** [comment]
Its not working with angular 14 version.

error NG6002: 'EmailEditorModule' does not appear to be an NgModule class.
